README.rst

```diff
@@ -42,18 +42,19 @@ You can now import the module `nimbleclient` and use it in any python script/app
 .. code-block:: pycon
 
     python3.7
-    Python 3.7.4 (default, Sep  7 2019, 18:27:02) 
+    Python 3.7.4 (default, Sep  7 2019, 18:27:02)
     [Clang 10.0.1 (clang-1001.0.46.4)] on darwin
     Type "help", "copyright", "credits" or "license" for more information.
-    >>> 
+    >>>
     >>> from nimbleclient import NimOSClient
     >>> import pprint
-    >>> client = NimOSClient("suneeth-vm1.vlab.nimblestorage.com", "admin", "admin")
+    >>> client = NimOSClient("1.1.1.1", "admin", "admin")
     >>>
-    >>> client.volumes.create("vol-1", size=10)
+    >>> client.volumes.create("vol-1", size=1024, description="Test volume1 for SDK")
```

Comment:
IMO having a consistent style of params is useful. This sample shows a positional param and key-value params together. I think we should always push for key-value params on the documentation.

Reply:
We need to mandate positional parameters like **create(name, ..), update(id, ..), delete(id, ..)** and **kwargs** are optional key-value pairs.

@cristian-medina Thoughts?